src/Chainweb/Chainweb/Configuration.hs

```diff
@@ -432,8 +433,8 @@ defaultChainwebConfiguration v = ChainwebConfiguration
     , _configLogGas = False
     , _configMinGasPrice = 1e-8
     , _configPactQueueSize = 2000
-    , _configReorgLimit = int defaultReorgLimit
-    , _configLocalRewindDepthLimit = int defaultLocalRewindDepthLimit
+    , _configReorgLimit = RewindLimit defaultReorgLimit
+    , _configLocalRewindDepthLimit = RewindLimit defaultLocalRewindDepthLimit
```

Comment:
Perhaps make `defaultReorgLimit` and `defaultLocalRewindDepthLimit` have type `RewindLimit` themselves?

Reply:
Nice catch, thanks!